src/parser/MaintainSentences.cpp

```diff
@@ -9,6 +9,55 @@
 
 namespace nebula {
 
+std::string SchemaPropItem::toString() const {
+    switch (propType_) {
+        case TTL_DURATION:
+            return folly::stringPrintf("ttl_duration = %ld",
+                                       boost::get<int64_t>(propValue_));
+        case TTL_COL:
+            return folly::stringPrintf("ttl_col = %s",
+                                       boost::get<std::string>(propValue_).c_str());
+        // TODO(YT) The following features will be supported in the future
+        case COMMENT:
```

Comment:
We'd better remove them before discussing what configurations we should support.

Reply:
Yeah，I will remove them.
Written here only identifies them as attributes of the schema